### DISCUSSION TITLE
How to fix an error when starting the service on a node for a web app

### DISCUSSION THREAD

**Discussion by gramunno** [gravitational/teleport#47032] [Q&A] [answered]
Hello,

I'm new to teleport.

I've installed the proxy/auth on a machine and it works fine.

I've created a token by executing the command 

sudo tctl tokens add --type=app --app-name=<app_name> --app-uri=http://<node_address>:<port>

The output of such command is the following 

teleport app start \
   --token=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXX \
   --ca-pin=sha256:YYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYYY \
   --auth-server=<tp_domain_FQDN>:443 \
   --name=<app_name>                     `# Change "<app_name>" to the name of your application.` \
   --uri=http://<node_address>:<port>              `# Change "http://<node_address>:<port>" to the address of your application.`

I executed the last command on the node where the web app is located but it fails with the following output:

2024-10-01T09:42:25Z INFO  Starting Teleport with a config file version:16.4.2 config_file:/etc/teleport.yaml common/teleport.go:756
ERROR: initialization failed
        configuration error
                config: cannot use both auth_server and proxy_server

Teleport version
Teleport v16.4.2 git:v16.4.2-0-ge414b8f go1.22.7

Surfing the Internet does not help.

Any idea?

Any help would be much appreciated.

Thanks in advance

**Comment by stevenGravy** [comment, marked as answer]
Looks like you have a `/etc/teleport.yaml` already on the machine. And you are specifying `auth-server` as a parameter. Is the proxy server the same as the auth server in `/etc/teleport.yaml`?  That parameter isn't required then.

**Reply by gramunno** [reply]
Thanks a lot!
Actually I have the configuration file already on the machine and I specified the auth-server parameter on the command line. By removing from the command executed from console the auth-server parameter the app service started successfully.
